src/plugins/intel_gpu/tests/test_cases/tile_gpu_test.cpp

```diff
@@ -249,3 +249,438 @@ TEST(tile_gpu, basic_in1x2x2x2_axis_z) {
         EXPECT_EQ(output_ptr[i], output_ref_ptr[i]) << "Index=" << i;
     }
 }
+
+namespace {
+template<typename T>
+struct Params {
+    tensor input_tensor;
+    std::vector<T> inputs;
+    std::vector<int64_t> repeats;
+    tensor output_tensor;
+    std::vector<T> outputs;
+    int output_axis;
+    int num_tiles;
```

Comment:
Where are output_axis and num_tiles used?

Reply:
done, removed